includes/scheduler/class-post.php

```diff
@@ -45,6 +49,8 @@ public static function transition_attachment_status( $post_id ) {
 				self::schedule_post_activity( 'trash', '', $post_id );
 				break;
 		}
+
+		\add_action( 'post_activitypub_add_to_outbox', array( self::class, 'send_announces' ), 10, 4 );
```

Comment:
@pfefferle Was this meant to be added here, rather than in `init()`? Or did I mess up a rebase?

Reply:
this was meant to be in the init!

Reply:
you messed it up ;)

Reply:
haha let me fix it!

Reply:
Oh, I messed that up in a merge somewhere, ha